### PR TITLE
Draft: Ntskrnl documentation

### PR DESCRIPTION
**Pull request by minecraftchest1**
A PR to track my documentation of Ntoskrnl. Jira issue: CORE-16149

Current Files
* ke/bug.c

Completed Files

Mirrored from [https://gitlab.com/minecraftchest1/reactos/-/merge_requests/1](https://gitlab.com/minecraftchest1/reactos/-/merge_requests/1)

### CODE REVIEW

File: ntoskrnl/ke/bug.c [hunk 1]
```diff
@@ -291,7 +346,13 @@ KeRosDumpStackFrames(IN PULONG_PTR Frame OPTIONAL,
...
 }
 
-CODE_SEG("INIT")
+/*
+ * @brief brief discription here.
+ *
+ * Full discription here.
+*/
+INIT_FUNCTION
+//CODE_SEG("INIT")
```
Comment (HBelusca) [inline ntoskrnl/ke/bug.c:355]:
```suggestion
/*
 * @brief brief discription here.
 *
 * Full discription here.
*/
CODE_SEG("INIT")
```

File: ntoskrnl/ke/bug.c [hunk 2]
```diff
@@ -622,7 +742,7 @@ KiDisplayBlueScreen(IN ULONG MessageId,
         InbvResetDisplay();
 
         /* Display blue screen */
-        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_BLUE);
+        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_GREEN);
```
Comment (HBelusca) [inline ntoskrnl/ke/bug.c:745]: Please revert.
```suggestion
        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_BLUE);
```

**Review by HBelusca — COMMENTED** [review]
Also, "description" is spelled with an 'e', not an 'i'.

**Comment by minecraftchest1** [comment]
> Also, "description" is spelled with an 'e', not an 'i'.
I'll fix that.
> And please squash all your commits together.
I think I'll keep them unsquashed until I figure out my build issues.

File: ntoskrnl/ke/bug.c [hunk 3]
```diff
@@ -291,6 +346,12 @@ KeRosDumpStackFrames(IN PULONG_PTR Frame OPTIONAL,
     }
 }
 
+/*
+ * @brief brief discription here.
+ *
+ * Full discription here.
+*/
+INIT_FUNCTION
```
Comment (learn-more) [inline ntoskrnl/ke/bug.c:354]: If you want to write documentation, why do you add code?

Comment (minecraftchest1) [reply ntoskrnl/ke/bug.c:354]: That was gitlab merging master into my branch.

**Comment by GeoB99** [comment]
Are you still working on this?

**Comment by minecraftchest1** [comment]
No.



Sent from my T-Mobile 4G LTE Device



-------- Original message --------
From: George Bișoc ***@***.***>
Date: 8/22/21 10:21 AM (GMT-06:00)
To: reactos/reactos ***@***.***>
Cc: Minecraftchest1 ***@***.***>, Author ***@***.***>
Subject: Re: [reactos/reactos] Draft: Ntskrnl documentation (#3376)


Are you still working on this?

**Comment by GeoB99** [comment]
"No", then as such I will close this PR for being stale.

**Comment by minecraftchest1** [comment]
That is fine with me.



Sent from my T-Mobile 4G LTE Device



-------- Original message --------
From: George Bișoc ***@***.***>
Date: 8/22/21 10:38 AM (GMT-06:00)
To: reactos/reactos ***@***.***>
Cc: Minecraftchest1 ***@***.***>, Author ***@***.***>
Subject: Re: [reactos/reactos] Draft: Ntskrnl documentation (#3376)


"No", then as such I will close this PR for being stale.

**Pull request closed without merging**